solidity/test/isms/OPStackIsm.t.sol

```diff
@@ -154,17 +149,15 @@ contract OPStackIsmTest is Test {
             (messageId)
         );
 
-        uint40 nonce = ICanonicalTransactionChain(L1_CANNONICAL_CHAIN)
```

Comment:
aren't we fork testing?

Reply:
we're not testing the nonce in the event